README.md

```diff
@@ -100,6 +100,20 @@ Create instance of `Socket` class by passing url of socketcluster-server end-poi
     socket.set_reconnection(false)
 ```
 
+- To set strategy for reconnection to server
+
+```ruby
```

Comment:
Not passing the reconnection strategy object to server?

Reply:
Just remove reconnection decay, we do not need it

Reply:
it should be same as  https://github.com/sacOO7/socketcluster-client-java

Reply:
I havent created the reconnection class for the strategy, Its a part of main socketcluster instance and reconnection module in used as mixin in socketcluster instance with reconnection related attributes